packages/manager/src/components/EmptyLandingPageResources/ResourcesLinksTypes.ts

```diff
@@ -10,6 +10,7 @@ export interface linkAnalyticsEvent {
 }
 
 export interface ResourcesHeaders {
+  dbLogo?: React.ReactNode;
```

Comment:
`ResourceHeaders` props are shared across all our empty-state landing pages. Could we name this more generically, here and in `ResourcesSection.tsx`, in case other landing pages make use of this prop in the future? `branding` was more generic - was there a reason for that change?

```suggestion
  logo?: React.ReactNode;
```